scripts/ci/validate-reborn-tests-workflow.py

```
@@ -0,0 +1,83 @@
+    )
+    return json.loads(raw)
+
+
+def reborn_package_matrix(metadata: dict) -> list[str]:
+    packages = []
+    for package in metadata["packages"]:
+        name = package["name"]
+        if (
+            name.startswith("ironclaw_reborn")
+            or name in {"ironclaw_product_workflow", "ironclaw_product_adapters"}
+            or name.startswith("ironclaw_webui_v2")
```

Ruff (0.15.17)

[warning] Call `startswith` once with a `tuple`

Merge into a single `startswith` call

(PIE810)